### ISSUE TITLE
The continue button is disabled all the time

### ISSUE THREAD

**Issue by ahangarha** [osmlab/onosm.org#139]
Regardless of the location, the "continue" button is always disabled. Even it I change the style and remove the `disabled` class, clicking on the button doesn't take me to the next tab.

![image](https://github.com/osmlab/onosm.org/assets/11241315/90cc2bc1-b391-4c1a-b5f0-b9737ba18bd4)

I tried different browsers but the result is all the same.

**Comment by ahangarha** [comment]
It seems one should search for the address first.

It is strange that the user is not allowed to directly pick the location on the map.

This might be a new feature but for sure this issue needs to get closed for now.

**Comment by iandees** [comment]
Yes, you need to search for an address or city name first and move the pin to the spot where the business is.

**Comment by jjiglesiasg** [comment]
Why you need to search for and address if you grant Access to your location? One pinpointed the contributor should be able to create the POI with his data. Or at least that's the way that sounds more quick and logic from the business contributor standpoint

**Comment by ArcaneDots** [comment]
The point is to start with address. Attempt to geocode it to find the
likely location. Then allow the user to move the pointer to the physical
location of the building.

This process make sure we get the official address for the business. It
also helps OSM to build a more realistic set of data. Making it easier for
geocoders and human map make subsequent guesses at nearby locations.

**Comment by iandees** [comment]
Also, your current location is not necessarily the location of the business.

**Comment by ahangarha** [comment]
I think it is better to let the user move the pointer regardless of whether they enter an address. The address can help the user to move the pointer faster (if needed).
